ktor-shared/ktor-serialization/ktor-serialization-jackson/jvm/src/JacksonWebsocketContentConverter.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2014-2021 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.serializaion.jackson
+
+import com.fasterxml.jackson.core.*
+import com.fasterxml.jackson.databind.*
+import com.fasterxml.jackson.module.kotlin.*
+import io.ktor.http.cio.websocket.*
+import io.ktor.serialization.*
+import io.ktor.util.reflect.*
+import io.ktor.utils.io.charsets.*
+import io.ktor.utils.io.core.*
+import io.ktor.utils.io.jvm.javaio.*
+import kotlinx.coroutines.*
+import java.io.*
+import kotlin.text.Charsets
+
+/**
+ * A jackson converter for the [WebSockets] plugin
+ */
+public class JacksonWebsocketContentConverter(private val objectmapper: ObjectMapper = jacksonObjectMapper()) :
+    WebsocketContentConverter {
+    override suspend fun serialize(charset: Charset, typeInfo: TypeInfo, value: Any): Frame {
+        val outputStream = ByteArrayOutputStream()
```

Comment:
There is no need to use input and output stream here, since frames are sent in whole, without streaming. You can use mapping to and from string directly